package/Style.roc

```diff
@@ -0,0 +1,56 @@
+module [Style, toCode]
+
+import Color exposing [Color]
+
+# Style (commonly known as Select Graphic Rendition or SGR) represents the control sequence for terminal display attributes.
+# It controls various attributes, which remain in effect until explicitly reset by a subsequent style sequence.
+# The provided attributes are common and well-supported, though not exhaustive.
+
+Style : [
```

Comment:
Can we shift this down so it's a doc comment, we will need to use `##` so it renders in the generated docs.

Reply:
done